### DISCUSSION TITLE
two Mi-V core inside PolarFire FPGA

### DISCUSSION THREAD

**Discussion by kevinwen2000** [Mi-V-Soft-RISC-V/miv-rv32-documentation#30] [Mi-V RV32 Soft Processors]
Hello,

Is that possible using two Mi-V RISC core within single PolarFire FPGA?

Don't know how to configure in this way.

Any comment on this are welcome.

Thanks
Kevin

**Comment by hughbreslin** [comment]
Hi @kevinwen2000 

Yes this is possible - it depends what you want to do, if they're independent (i.e don't have any overlap or need to communicate with each other) you can just drag and drop two cores from the catalog in the tools and add in peripherals and connections for them independently. If they need to communicate you will need to set up a shared memory between them. 

The only thing I can think of is debug where you will need to set individual target IDs for the cores so they can be accessed individually by the debugger.

If you can give more details of what you want to achieve I could maybe help more :)
